files/web.xml

```diff
@@ -0,0 +1,1678 @@
+        <url-pattern>/x/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>file-server</servlet-name>
+        <url-pattern>/download/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>status-servlet</servlet-name>
+        <url-pattern>/status</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>xmlrpc</servlet-name>
+        <url-pattern>/rpc/xmlrpc</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>trackback</servlet-name>
+        <url-pattern>/rpc/trackback/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>servlet-module-container-servlet</servlet-name>
+        <url-pattern>/plugins/servlet/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>labels</servlet-name>
+        <url-pattern>/label/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>jcaptcha</servlet-name>
+        <url-pattern>/jcaptcha/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>exportword</servlet-name>
+        <url-pattern>/exportword</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>noop</servlet-name>
+        <url-pattern>/s/*</url-pattern>
+    </servlet-mapping>
+
+    <!--
+    Noop filter mapping for the trusted app certificate which is serviced exclusively by the Seraph trusted app filter.
+    This servlet mapping is necessary for Websphere 6.1 which still likes to forward the request to a servlet even if
+    the filter chain is not fully followed.
+    -->
+    <servlet-mapping>
+        <servlet-name>noop</servlet-name>
+        <url-pattern>/admin/appTrustCertificate</url-pattern>
+    </servlet-mapping>
+
+    <!--
+    As the REST module is implemented using only a filter this noop servlet is required (CONF-17578)
+    -->
+    <servlet-mapping>
+        <servlet-name>noop</servlet-name>
+        <url-pattern>/rest/*</url-pattern>
+    </servlet-mapping>
+
+    <session-config>
+        <session-timeout>60</session-timeout>
+        <tracking-mode>COOKIE</tracking-mode>
+    </session-config>
+
+    <welcome-file-list>
+        <welcome-file>default.jsp</welcome-file>
+        <welcome-file>index.action</welcome-file>
+    </welcome-file-list>
+
+    <!-- redirect all 500 errors to confluence error page -->
+    <error-page>
+        <error-code>500</error-code>
+        <location>/500page.jsp</location>
+    </error-page>
+
+    <error-page>
+        <error-code>404</error-code>
+        <location>/fourohfour.action</location>
+    </error-page>
+    <resource-ref>
+        <description>Connection Pool</description>
+        <res-ref-name>jdbc/confluence</res-ref-name>
+        <res-type>javax.sql.DataSource</res-type>
+        <res-auth>Container</res-auth>
+    </resource-ref>
+</web-app>
```

Comment:
Is all this guaranteed to stay the same from version to version? Perhaps it would be better to only support augeas-based configuration of the values instead of having a huge static file like this. :/